src/plugins/intel_gpu/src/kernel_selector/kernel_selector_params.h

```diff
@@ -131,6 +131,7 @@ class ParamsKey {
                 uint32_t asym_d_quantization : 1;
                 uint32_t dynamic_shapes : 1;
                 uint32_t compressed_weights : 1;
+                uint32_t single_batch_opt : 1;
```

Comment:
I think we can remove this from common params, and use only FC specific `_single_batch_optimized`

Reply:
removed.